### PR TITLE
chore: update built-in metrics description

### PR DESCRIPTION
**Pull request by heitortsergent**
## What?

Update some of the descriptions for the Web Vital Metrics, and reorder the tables on that page.

## Checklist

Please fill in this template:
- [x] I have used a meaningful title for the PR.
- [x] I have described the changes I've made in the "What?" section above.
- [x] I have performed a self-review of my changes.
- [x] I have run the `make docs` command locally and verified that the changes look good.

If updating the documentation for the most recent release of k6: 
- [x] I have made my changes in the `docs/sources/next` folder of the documentation.
- [x] I have reflected my changes in the `docs/sources/v{most_recent_release}` folder of the documentation.
- [x] I have reflected my changes in the relevant (*e.g.*  when correcting a documentation error) folders of the previous k6 versions of the documentation.

### CODE REVIEW

File: docs/sources/v0.48.x/using-k6/metrics/reference.md [hunk 1]
```diff
@@ -13,16 +13,16 @@ Each supported protocol also has its specific metrics.
...
+| vus                | Gauge   | Current number of active virtual users                                                                                                                                                                                                                                                        |
+| vus_max            | Gauge   | Max possible number of virtual users (VU resources are [pre-allocated](https://grafana.com/docs/k6/<K6_VERSION>/using-k6/scenarios/concepts/arrival-rate-vu-allocation), to avoid affecting performance when scaling up load )                                                                |
+| iterations         | Counter | The aggregate number of times the VUs execute the JS script (the `default` function).                                                                                                                                                                                                         |
```
Comment (ppcano) [inline docs/sources/v0.48.x/using-k6/metrics/reference.md:25]: If this PR sort by name, then `iterations` should be before `iteration_duration`.

File: docs/sources/v0.48.x/using-k6/metrics/reference.md [hunk 2]
```diff
@@ -33,19 +33,19 @@ These metrics are generated only when the test makes HTTP requests.
...
-| http_req_receiving       | Trend   | Time spent receiving response data from the remote host. `float`                                                                                                                                                                             |
-| http_req_duration        | Trend   | Total time for the request. It's equal to `http_req_sending + http_req_waiting + http_req_receiving` (i.e. how long did the remote server take to process the request and respond, without the initial DNS lookup/connection times). `float` |
-| http_req_failed          | Rate    | The rate of failed requests according to [setResponseCallback](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-http/set-response-callback).                                                                                                       |
+| http_reqs                | Counter | How many total HTTP requests k6 generated.                                                                                                                                                                                                   |
```
Comment (ppcano) [inline docs/sources/v0.48.x/using-k6/metrics/reference.md:48]: `http_req_failed` has been deleted and is unavailable now. Please, restore it.

**Review by ppcano — CHANGES_REQUESTED** [review]
Two changes have been requested.

Additionally, there are conflicting files to be resolved before this can be merged.

I am unsure whether it is necessary to update old API versions.

File: docs/sources/v0.48.x/using-k6/metrics/reference.md (hunk 2 above)
Comment (heitortsergent) [reply docs/sources/v0.48.x/using-k6/metrics/reference.md:48]: `http_req_failed` is still listed on line 43.

Comment (ppcano) [reply docs/sources/v0.48.x/using-k6/metrics/reference.md:48]: apologies

**Review by ppcano — APPROVED** [review]

File: docs/sources/v0.48.x/using-k6/metrics/reference.md (hunk 2 above)
Comment (heitortsergent) [reply docs/sources/v0.48.x/using-k6/metrics/reference.md:48]: no worries! 🙏

**Comment by heitortsergent** [comment]
Thanks for the review @ppcano 🙇 .

**Pull request merged**